coffee-chats/src/main/java/com/google/step/coffee/servlets/GroupMembersServlet.java

```diff
@@ -42,17 +46,28 @@ public Object post(JsonServletRequest request) throws IOException, HttpError {
     User user = User.builder()
         .setId(request.getRequiredParameter("user"))
         .build();
-    GroupMembership.Status status;
+    GroupMembership.Status status, oldStatus;
 
     try {
       status = GroupMembership.Status.valueOf(request.getRequiredParameter("status"));
     } catch (IllegalArgumentException exception) {
       throw new HttpError(HttpServletResponse.SC_BAD_REQUEST, "Invalid value for parameter 'status'");
     }
 
+    oldStatus = groupStore.getMembershipStatus(group, user);
+
     PermissionChecker.ensureCanUpdateMembershipStatus(group, user, status);
     groupStore.updateMembershipStatus(group, user, status);
 
+    if (oldStatus == GroupMembership.Status.NOT_A_MEMBER || status == GroupMembership.Status.NOT_A_MEMBER) {
+      // somebody joined or left the group
```

Comment:
What if organizer of the event left?

Reply:
The group owner can't leave the group :) Ideally of course we would have a service account for the app, so that this wouldn't be a problem anyway.

Reply:
If administrator (not owner) created a meeting, it's still group owner who is organizer of the meeting?

Reply:
Yeah